src/pages/projects/index.page.tsx

```diff
@@ -58,11 +62,8 @@ export default function Projects() {
           return defaultTab
       }
     })
-  }, [userAddress, router.query])
-
-  const [searchText, setSearchText] = useState<string>(
-    (router.query.search as string | undefined) ?? '',
-  )
+    return setSearchText(router.query.search as string)
```

Comment:
maybe this can just be called instead of returned?

Reply:
I was getting a typescript error earlier without the return. let me remove it and see if i'm still getting it.